apisix/plugins/file-logger.lua

```diff
@@ -14,23 +14,23 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
-local log_util     =   require("apisix.utils.log-util")
-local core         =   require("apisix.core")
-local ngx          =   ngx
-local io_open      =   io.open
+local log_util = require("apisix.utils.log-util")
+local core = require("apisix.core")
+local ngx = ngx
+local io_open = io.open
 local is_apisix_or, process = pcall(require, "resty.apisix.process")
 
-
 local plugin_name = "file-logger"
-
+local std_out_file = "/dev/stdout"
 
 local schema = {
     type = "object",
     properties = {
         path = {
             type = "string"
         },
-        include_resp_body = {type = "boolean", default = false},
+        log_to_stdout = { type = "boolean", default = false },
```

Comment:
Would it be better to use a special value to indicate the stdout like Nginx (https://nginx.org/en/docs/ngx_core_module.html#error_log)?
So we don't need an extra configuration to shadow the path.

Reply:
Thanks for catching that, I've already updated it.